Updated tipuesearch to v5

phips opened the pull request:
For some reason I was getting links as 'undefined' for my searches until I upgraded the tipuesearch JS to version 5. It now works (see probably.co.uk)

Commit:
Updated tipuesearch to v5

cranmer commented:
Hello

I have the same issue. It started shortly after 
9ec749e <https://github.com/DandyDev/pelican-bootstrap3/commit/9ec749e38d2a150537391340c73f411c4d473984>

Kyle

> On Jun 18, 2015, at 11:23 AM, Mark Phillips <notifications@github.com> wrote:
> 
> For some reason I was getting links as 'undefined' for my searches until I upgraded the tipuesearch JS to version 5. It now works (see probably.co.uk)
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/DandyDev/pelican-bootstrap3/pull/195 <https://github.com/DandyDev/pelican-bootstrap3/pull/195>
> Commit Summary
> 
> Updated tipuesearch to v5
> File Changes
> 
> M static/tipuesearch/tipuesearch.css <https://github.com/DandyDev/pelican-bootstrap3/pull/195/files#diff-0> (131)
> M static/tipuesearch/tipuesearch.js <https://github.com/DandyDev/pelican-bootstrap3/pull/195/files#diff-1> (314)
> M static/tipuesearch/tipuesearch.min.js <https://github.com/DandyDev/pelican-bootstrap3/pull/195/files#diff-2> (169)
> M static/tipuesearch/tipuesearch_content.js <https://github.com/DandyDev/pelican-bootstrap3/pull/195/files#diff-3> (26)
> M static/tipuesearch/tipuesearch_set.js <https://github.com/DandyDev/pelican-bootstrap3/pull/195/files#diff-4> (58)
> Patch Links:
> 
> https://github.com/DandyDev/pelican-bootstrap3/pull/195.patch <https://github.com/DandyDev/pelican-bootstrap3/pull/195.patch>
> https://github.com/DandyDev/pelican-bootstrap3/pull/195.diff <https://github.com/DandyDev/pelican-bootstrap3/pull/195.diff>
> —
> Reply to this email directly or view it on GitHub <https://github.com/DandyDev/pelican-bootstrap3/pull/195>.
>

sethfischer committed:
Check tag_cloud plugin is enabled

In version 3.6.0 of Pelican tag_cloud was moved from core to a plugin.
Hence we check the plugin is enabled before displaying the tag cloud.

DandyDev commented:
Hi!

First of all, thanks for your pull request! I appreciate the contribution, but I cannot accept this PR at the moment. It's with heavy heart that I have decided that I can no longer maintain this theme due to lack of time and changing interests. To makes sure the theme lives on however and not goes to waste, I've donated it to the [Pelican community](https://github.com/getpelican/pelican-themes). It can now be found in the official [pelican-themes repository](https://github.com/getpelican/pelican-themes/tree/master/pelican-bootstrap3). I'd suggest you re-submit your pull request there.

Sorry for the inconvenience!
